cmd/delete.go

```diff
@@ -4,6 +4,7 @@ import (
 	"strings"
 
 	"github.com/pkg/errors"
+	analytics "github.com/segmentio/analytics-go"
```

Comment:
I recommend using https://github.com/segmentio/analytics-go/tree/v3.0. We've already deprecated v2 and are just waiting for until the deprecation date to merge v3.0 into master.

Reply:
yup! I'm using v3 :)

Reply:
ah 👍